fiddler/requirements.in

```diff
@@ -0,0 +1 @@
+fiddler-client==1.4.0
```

Comment:
```suggestion
```
Not actually needed here since the agent isn't going to be built with the package.